README.md

```diff
@@ -813,6 +834,25 @@ You can specify the association of the server profile with each of the resources
 
 - **\<network_type\>_connections** (Hash) Optional - Specify connections with the desired resource type. The Hash should have `<network_name> => <connection_data>` associations. See the [examples](examples/server_profile.rb) for more information.
 - **os_deployment_plan** (String) Optional - Specify the OS Deployment Plan to be applied with the Server Profile. The OS Deployment Plan needs to be created in Image Streamer appliance in order to appear in OneView. See the [example](examples/image_streamer/server_profile_deploy.rb) for more information.
+- **volume_attachments** (Array<Hash>) Optional - Specify a list of volume attachments to be created when create/update the Server Profile. See the [example](examples/server_profile.rb) for more information.
+
+  To attach a Volume already created, put into the 'volume_attachments' something like:
+  ```ruby
+  {
+    volume: 'test2', # name of existent Oneview Volume
+    attachment_data: {} # key-pair data to be the specific attributes of the Oneview Volume Attachment
```

Comment:
Is this intended to be empty?

If not, but you want to ommit the data use something like `attachment_data: { ... }`